scout/server/blueprints/variants/controllers.py

```diff
@@ -43,6 +44,21 @@
 LOG = logging.getLogger(__name__)
 
 
+def get_variants_page(form=ImmutableMultiDict) -> int:
```

Comment:
👍

Reply:
Does change current ops a little with not checking args.page from an initial GET, but afaik that should never want anything other than page 1 anyway - so good riddance!

Reply:
Exactly, though before it was technically possible to ask for another page via get. Though I do not think we use that in code anymore, since a looong time really.